helm/common/utils.sh

```diff
@@ -93,7 +93,7 @@ function run_helm_unittest() {
     fi
 
     # Run a Helm unit test
-    helm unittest . --helm3
+    helm unittest .
```

Comment:
@szh Because we're pinning the `helm-unittest` version, we need to keep the `--helm3` flag. I'll push a commit updating this.